app/views/journeys/income/IncomeCYAView.scala.html

```diff
@@ -0,0 +1,38 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import templates.helpers.Heading
+
+@this(layout: templates.Layout, govukButton: GovukButton, heading: Heading, govukSummaryList: GovukSummaryList)
+
+@(taxYear: Int, summaryList: SummaryList, nextRoute: String, authUserType: String)(implicit request: Request[_], messages: Messages)
+
+@layout(pageTitle = titleNoForm(messages(s"checkYourIncome.title.$authUserType"))) {
+
+    @heading(
+        heading = messages(s"checkYourIncome.title.$authUserType"),
+        subheading = Some(messages("site.taxYearCaption", (taxYear - 1).toString, taxYear.toString)),
+        extraClasses = "govuk-!-margin-bottom-3"
+    )
+
+    @govukSummaryList(summaryList)
+
+    <p class="govuk-body">
```

Comment:
I think for flexible formatting the button should be in a div that matches the prototype classes, rather than in a paragraph.
See: https://github.com/hmrc/income-tax-self-employment-frontend/pull/30/files#diff-2ee5d34f2262180db613260ddee84fa56f3539a084aa7c25475a789560f7d379